examples/camera-controller/webrtc_manager/WebRTCProviderClient.h

```diff
@@ -119,6 +129,17 @@ class WebRTCProviderClient : public chip::app::CommandSender::Callback
         kEndSession           = 5,
     };
 
+    enum class State : uint8_t
+    {
+        Idle,             ///< Default state, no communication initiated yet
+        Connecting,       ///< Waiting for OnDeviceConnected or OnDeviceConnectionFailure callbacks to be called
+        AwaitingResponse, ///< Waiting for command response from camera
+        AwaitingAnswer,   ///< Waiting for Answer command from camera
```

Comment:
After the Answer is received and the controller responds successfully, it should transition to a Stable state, e.g., HaveRemoteAnswer.

Reply:
Yes, after Answer message is handled, we go back to Idle state

```
void WebRTCProviderClient::NotifyRemoteDecryptorReceived(uint16_t webRTCSessionID)
{
    ChipLogProgress(Camera, "Remote decryptor received for WebRTC session ID: %u", webRTCSessionID);

    DeviceLayer::SystemLayer().CancelTimer(OnSessionEstablishTimeout, this);
    MoveToState(State::Idle);
}
```